exec/java-exec/src/main/java/org/apache/drill/exec/store/ischema/InfoSchemaRecordGenerator.java

```diff
@@ -379,7 +379,7 @@ public Views(OptionManager optionManager) {
     public boolean visitTable(String schemaName, String tableName, Table table) {
       if (table.getJdbcTableType() == TableType.VIEW) {
         records.add(new Records.View(IS_CATALOG_NAME, schemaName, tableName,
-                    ((DrillViewInfoProvider) table).getViewSql()));
+            table instanceof DrillViewInfoProvider ? ((DrillViewInfoProvider) table).getViewSql() : ""));
```

Comment:
Could you please add a comment that view SQL is not available for the case of some non-Drill views, for example, JDBC view.

Reply:
Done.